BlueprintUI/Sources/Layout/Builder/StackElement+Builder.swift

```diff
@@ -0,0 +1,115 @@
+import UIKit
+
+/// `StackChild` is a wrapper which allows an element to define its StackLayout traits and Keys.
+/// This struct is particularly useful when working with `@StackElementBuilder`.
+/// By default, elements will default to a nil key and the default `StackLayout.Traits` initializer.
+/// `@StackElementBuilder` will check every child to see if it can be type cast to a `StackChild`
+/// and then pull of the given traits and key and then apply those to the stack
+public struct StackChild: ProxyElement {
+    public var traits: StackLayout.Traits
+    public var key: AnyHashable?
+    private let wrappedElement: Element
+
+    public init(
+        wrappedElement: Element,
```

Comment:
Nit: Just `wrapping` for this – Swift style is to leave off the value type (`Element`), since that is clear from the type itself